src/types/interactions/application_command_callback_data.ts

```diff
@@ -0,0 +1,21 @@
+import { Embed } from "../../embeds/embed.ts";
+import { AllowedMentions } from "../../messages/allowed_mentions.ts";
+import { SnakeCaseProps } from "../../util.ts";
+
+export interface ApplicationCommandCallbackData {
```

Comment:
Rename to `InteractionApplicationCommandCallbackData`
```suggestion
export interface InteractionApplicationCommandCallbackData {
```